README.md

```diff
@@ -103,7 +103,7 @@ We are now at or near 100% compatibility with JSCS. If you try ESLint and believ
 
 ### Does Prettier replace ESLint?
 
-No, ESLint does both traditional linting (looking for problematic patterns) and style checking (enforcement of conventions). You can use ESLint for everything, or you can combine both using Prettier to format your code and ESLint to catch possible errors.
+No, ESLint and Prettier have diffent jobs: ESLint is a linter (looking for problematic patterns) and Prettier is a code formatter. Using both tools is common, refer to [Prettier's documentation](https://prettier.io/docs/en/install#eslint-and-other-linters) to learn how to configure them to work well with each other.
```

Comment:
Maybe we could mention that ESLint can also enforce code formatting through custom rules? Like

```suggestion
No, ESLint and Prettier have diffent jobs: ESLint is a linter that can be extended (mainly looking for problematic patterns, but also checking styles with custom rules) and Prettier is an opinionated code formatter. Using both tools is common, refer to [Prettier's documentation](https://prettier.io/docs/en/install#eslint-and-other-linters) to learn how to configure them to work well with each other.
```

Reply:
It seems that it's not what's officially recommended though. ESLint recommends to [« _use a source code formatter instead_ »](https://eslint.org/blog/2023/10/deprecating-formatting-rules/).